systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/MultipleListenersST.java

```diff
@@ -158,15 +159,11 @@ void testCombinationOfEveryKindOfListener(ExtensionContext extensionContext) thr
         runListenersTest(extensionContext, multipleDifferentListeners, clusterName);
     }
 
-    private void runListenersTest(ExtensionContext extensionContext, List<GenericKafkaListener> listeners, String clusterName) throws Exception {
-
+    private void runListenersTest(ExtensionContext extensionContext, List<GenericKafkaListener> listeners, String clusterName) {
         LOGGER.info("This is listeners {}, which will verified.", listeners);
 
         // exercise phase
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 3)
-            .editMetadata()
-                .withNamespace(namespace)
-            .endMetadata()
```

Comment:
This explicit setting of namespace has to be here. Because in some cases the Kubernetes client may not be in `multi-list-st` namespace and could cause a problem.